### PR TITLE
Fix i18n of raster layer in the profile

### LINKED ISSUE #918: Translation of Raster Layer Name Key
https://github.com/camptocamp/c2cgeoportal/blob/master/doc/integrator/raster.rst states the terms mns and mnt are simply translatable by the client.

I tried to translate them in /project/static/js/Proj/Lang/de.js, but this doesn't help.

Example: http://mapfish-geoportal.demo-camptocamp.com/eleu-demo/wsgi/?lang=de still has the term "mnt" instead of the translation "DTM".

1. Should they be translated in the plugin / CGXP?
2. Will the mnt also be translated in the exported CSV?

### PR DESCRIPTION
**Pull request by sbrunner**
fix #918

### CODE REVIEW

File: core/src/script/CGXP/plugins/Profile.js [hunk 1]
```diff
@@ -227,6 +227,11 @@ cgxp.plugins.Profile = Ext.extend(gxp.plugins.Tool, {
     /** private: method[addActions]
      */
     addActions: function() {
+        this.i18nRasterLayers = [];
+        Ext.each(this.rasterLayers, function(layer) {
+            this.i18nRasterLayers.push(OpenLayers.i18n(layer));
+        }, this);
```
Comment (asaunier) [inline core/src/script/CGXP/plugins/Profile.js]: What about using https://developer.mozilla.org/fr/docs/Web/JavaScript/Reference/Objets_globaux/Array/map instead?

File: core/src/script/CGXP/plugins/Profile.js [hunk 2]
```diff
@@ -533,15 +535,14 @@ cgxp.plugins.Profile = Ext.extend(gxp.plugins.Tool, {
...
         var datum = this.data[row];
         var point = new OpenLayers.Geometry.Point(datum.x, datum.y);
         this.marker && this.marker.destroy();
 
         var label = [datum.dist + " m"];
-        for (var i=0; i < layers.length; i++) {
-            var l = layers[i];
-            label.push(l + " : " + datum[this.valuesProperty][l] + " m");
+        for (var i = 0 ; i < this.rasterLayers.length ; i++) {
+            label.push(this.i18nRasterLayers[i] + " : " +
```
Comment (asaunier) [inline core/src/script/CGXP/plugins/Profile.js]: Depending on the language, there might be no blank space before ":"...

File: core/src/script/CGXP/plugins/Profile.js [hunk 3]
```diff
@@ -533,15 +535,14 @@ cgxp.plugins.Profile = Ext.extend(gxp.plugins.Tool, {
...
+                    datum[this.valuesProperty][this.rasterLayers[i]] + " m");
```
Comment (asaunier) [inline core/src/script/CGXP/plugins/Profile.js]: Are data always in meters?

File: core/src/script/CGXP/plugins/Profile.js (hunk 1 above)
Comment (sbrunner) [inline core/src/script/CGXP/plugins/Profile.js]: Much better but don't work on ie8 :-(
http://kangax.github.io/compat-table/es5/#Array.prototype.map

File: core/src/script/CGXP/plugins/Profile.js (hunk 2 above)
Comment (sbrunner) [inline core/src/script/CGXP/plugins/Profile.js]: Good question,
Actually in french we add it and it's the most used language ...
I can change to make it internationalisable :-)

File: core/src/script/CGXP/plugins/Profile.js (hunk 3 above)
Comment (sbrunner) [inline core/src/script/CGXP/plugins/Profile.js]: Actually yes! :-)

File: core/src/script/CGXP/plugins/Profile.js (hunk 1 above)
Comment (asaunier) [inline core/src/script/CGXP/plugins/Profile.js]: Do we really need to support IE8?
I thought IE9 support was enough?

**Comment by asaunier** [comment]
A few minor comments, else it looks good to me.

File: core/src/script/CGXP/plugins/Profile.js (hunk 1 above)
Comment (sbrunner) [inline core/src/script/CGXP/plugins/Profile.js]: I didn't have any clear question about the usage of IE8, then I prefer to be copatible with him.

**Pull request merged**